data/repos.yml

```diff
@@ -590,21 +764,61 @@
   team: "#govuk-platform-reliability-team"
   type: Utilities
 
+# Same support model as govuk-user-reviewer, i.e. _really_ just maintained by 2nd line / leads
+# I'll probably add this in a separate PR.
+- repo_name: govuk-rota-generator
+  team: "#govuk-platform-reliability-team"
+  type: Utilities
+  sentry_url: false
+  dashboard_url: false
+
 - repo_name: govuk-rfcs
   team: "#govuk-developers"
   type: Utilities
 
+# Check with Roch to see if this is still used
+# (fork with 1 extra commit: https://github.com/alphagov/govuk-ruby/commit/07a14ea112ada861c95efb1ac55881c51f0e830e)
+# I can't see a corresponding commit in govuk-puppet
+# This will go away with Replatforming though, so is ripe for archiving soon anyway.
+- repo_name: govuk-ruby
+
+# Actively used, should be added to this list
+- repo_name: govuk-ruby-images
+  type: Utilities
+  team: "#govuk-replatforming"
+  sentry_url: false
+  dashboard_url: false
+
 - repo_name: govuk-rota-announcer
   private_repo: true
   team: "#govuk-platform-reliability-team"
   type: Utilities
 
+# Looks actively used
+- repo_name: govuk-s3-mirror
+  private_repo: true # internal
+  team: "#data-products"
+  type: Data science
+  sentry_url: false
+  dashboard_url: false
```

Comment:
@nacnudus

Reply:
This is actively used by Data Services, but it's important for GOV.UK to know about it too.  How can I go about documenting it?

Reply:
Thanks @nacnudus , please raise a PR to this repo, adding the repo to this YAML file like in this PR. Feel free to add a `  `description` property to document it ([example](https://github.com/alphagov/govuk-developer-docs/blob/31594c6a472b5b52624936576e9411dd958fcae4/data/repos.yml#L454-L455)) but it's not mandatory.

The important thing is that it's in the list of GOV.UK repos and that it has an owner (denoted by `team`).